cypress/e2e/gui/basic.cy.js

```diff
@@ -0,0 +1,96 @@
+describe('Basic', () => {
+  beforeEach(() => {
+    Cypress.on('window:before:load', window => {
+      window.document.cookie = 'cookieConsent=accepted'
+    })
+    cy.visit('/')
+  })
+
+  it('keeps the filters when coming back from the customer details view', () => {
+    cy.get('[data-testid="size-filter"]').select('Small')
+    cy.get('[data-testid="industry-filter"]').select('Finance')
+    cy.contains('button', 'View').click()
+    cy.contains('h2', 'Customer Details').should('be.visible')
+    cy.contains('button', 'Back').click()
+    cy.contains('h1', 'EngageSphere').should('be.visible')
+    cy.get('[data-testid="size-filter"]').should('have.value', 'Small')
+    cy.get('[data-testid="industry-filter"]').should('have.value', 'Finance')
+  })
```

Comment:
```suggestion
  })

```

Para melhorar a legibilidade, é recomendado deixar uma linha em branco entre cada bloco `it`, para visualmente tornar mais clara a separação entre cada caso de teste.

> O mesmo vale para cada bloco `it` abaixo.